providers/vic/proxy/isolation_proxy.go

```diff
@@ -36,8 +36,11 @@ type IsolationProxy interface {
 	UnbindScope(op trace.Operation, handle string, name string) (string, interface{}, error)
 
 	Handle(op trace.Operation, id, name string) (string, error)
-	State(op trace.Operation, id, name string) (string, error)
 	Remove(op trace.Operation, id string, force bool) error
+
+	State(op trace.Operation, id, name string) (string, error)
+	PodConfig(op trace.Operation, id, name string) (*models.ContainerConfig, error)
```

Comment:
This is going to increase tech debt.  We shouldn't return swagger models outside of the isolation proxy.  This is one of our complaints about Swagger.  It doesn't hide the data marshalling.  Can we create new data structures?  It sucks but it's the only way to keep the code that uses the isolation proxy clean.  I was unsuccessful at keeping people from doing this to the persona on VIC, but since it's just two of us, we can do better.